src/radical/pilot/task_manager.py

```diff
@@ -15,8 +15,9 @@
 from . import states    as rps
 from . import constants as rpc
 
-from .task_description import TaskDescription, RAPTOR_MASTER, RAPTOR_WORKER
+from .task_description import RAPTOR_MASTER, RAPTOR_WORKER
 from .raptor_tasks     import RaptorMaster, RaptorWorker
+from .resource_config  import Slot
```

Comment:
seems it is not used here, could be removed

Reply:
ack